code/game/machinery/flasher.dm

```diff
@@ -29,7 +29,7 @@
 	base_state = "pflash"
 	density = TRUE
 
-/obj/machinery/flasher/portable/ComponentInitialize()
+/obj/machinery/flasher/portable/Initialize()
```

Comment:
Could you make these have the mapload argument just for consistency? 

I know the var isn't used anywhere, but readability would be nice.

Reply:
Yeah I can do do that.